### ISSUE TITLE
Run Above/Below Cells should be disabled while debugging inside of cell

### ISSUE THREAD

**Issue by iemejia** [microsoft/vscode#161815]
## Environment data

**VS Code Environment:**
Version: 1.72.0-insider
Commit: b9f7f85e90b4d51b38fcbf4232801ab04bd659cb
Date: 2022-09-23T08:15:07.186Z
Electron: 19.0.17
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 21.6.0
Sandboxed: Yes

**VS Code Extensions**
Jupyter Extension: v2022.9.1002681037 Pre-Release
Python Extension: v2022.15.12631011 Pre-Release
Azure Machine Learning - Remote: v0.19.2022092309 Pre-Release
Azure Data Lake Tools: v0.2.15

**Python Environment**
Conda: 4.13.0
Python: 3.9.13
Type of virtual environment: Conda + IPyKernel

**.net Environment**
.net: 6.0.401

**Operating System**
macOS Monterrey 12.6
Macbook Air (M1, 2020)

## Expected behaviour

Disable the run above/below cells during debugging so users don't get confused by 'run' like icons.

## Actual behaviour

When a user is debugging a cell in a Jupyter Notebook the arrows to run above and below cells are still enabled and the user can easily confused them as step-next or continue buttons (being part of the debug cell.

Additionally if the user clicks on the run above/below buttons the action is 'queued' and executed after the debugging of the cell is over, which is an 'unexpected' annoyance.

## Steps to reproduce:

1. Create a notebook with multiple cells
2. Debug code on one of the cell in the middle
3. Press the run below button during the debugging of the cell

Buttons available and confusing

![reportbug-azuremlremote9](https://user-images.githubusercontent.com/79476/192248052-0a268f07-8b82-4242-90c2-d055f0c73203.gif)

Once the debug of the cell is over the Run Above/Below action executes restarting other cells
![reportbug-azuremlremote10](https://user-images.githubusercontent.com/79476/192248208-237857b0-d5a3-407b-986e-46a6c9543b9f.gif)

**Comment by roblourens** [comment]
Hm, I actually can't really do this perfectly since "debugging a notebook" is an extension concept, but the buttons are managed by core. I could disable them when paused in some cell in the notebook but then they would blink as you step. Maybe if you pause in the notebook, they get disabled until that debug session ends? I can explore later.

**Comment by iemejia** [comment]
Yes just being disabled until the debug session finishes would be already an improvement, maybe a bit confusing but at least the user won't end up re-running the full notebook by mistake.

**Comment by iemejia** [comment]
I think I accidentally created this issue here instead of in https://github.com/microsoft/vscode-jupyter/
Can someone please move it there if it is the right place for this one to be. Thanks
